test/specs/mainnet/01_TransferringFunds/Optimism.spec.js

```diff
@@ -71,7 +72,13 @@ describe('The PrimeSDK, when transfer a token with optimism network on the MainN
       );
     }
 
-    let output = await optimismMainNetSdk.getAccountBalances({
+    // initializating Data service...
+    optimismDataService = new DataUtils(
+      process.env.PROJECT_KEY,
+      graphqlEndpoints.QA,
```

Comment:
Mainnets are supposed to be graphqlEndpoints.PROD and QA is for only testnets

Reply:
Updated the code for all the mainnet test cases.